website/docs/r/schema_site_anp_epg_static_port.html.markdown

```diff
@@ -30,6 +32,46 @@ resource "mso_schema_site_anp_epg_static_port" "static_port" {
 }
 ```
 
+### path_type: vpc ###
+
+```hcl
+resource "mso_schema_site_anp_epg_static_port" "static_port" {
+  schema_id = "5c4d5bb72700000401f80948"
+  site_id = "5c7c95b25100008f01c1ee3c"
+  template_name = "Template1"
+  anp_name = "ANP"
+  epg_name = "DB"
+  path_type = "port"
+  deployment_immediacy = "lazy"
+  pod = "pod-1"
+  leaf = "101-102"
+  path = "HX-FI-B_PolGrp"
+  vlan = 200
+  mode = "untagged"
+
+}
+```
+
+### path_type: dpc
+
+```hcl
+resource "mso_schema_site_anp_epg_static_port" "static_port" {
+  schema_id = "5c4d5bb72700000401f80948"
+  site_id = "5c7c95b25100008f01c1ee3c"
+  template_name = "Template1"
+  anp_name = "ANP"
+  epg_name = "DB"
+  path_type = "dpc"
+  deployment_immediacy = "lazy"
+  pod = "pod-7"
+  leaf = "109"
+  path = "eth1/10"
```

Comment:
dpc is also a port group name here not an interface.

Reply:
ok